packages/okta-react/src/SecureRoute.js

```diff
@@ -14,6 +14,32 @@ import React, { Component } from 'react';
 import { Route } from 'react-router';
 import withAuth from './withAuth';
 
+
+class RenderWrapper extends Component {
+  checkAuthentication() {
+    if (this.props.authenticated === false) {
+      this.props.login();
```

Comment:
Do we need a `fromUri` value here?

Reply:
We default to the current location if we don't pass anything:

https://github.com/okta/okta-oidc-js/blob/master/packages/okta-react/src/Auth.js#L84

This actually seems to be the safest approach, because the router does udpate the window location before hitting this wrapped render, and the route path can be a matching expression:

https://reacttraining.com/react-router/web/api/Route/path-string

So by dynamically pulling the window location at this time we're likely to get the actual path that the user should be on after they are sent back 👍